rke/k8s_version_info.go

```diff
@@ -181,7 +181,7 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
 		// This version includes nodelocal dns only available in RKE v1.0.7 and up
-		"v1.17.6-rancher2-3": {
+		"v1.17.8-rancher1-1": {
 			MinRancherVersion: "2.4.6-rc0",
```

Comment:
min Rancher version should be 2.4.4, we are making this available to 2.4.4 also.